sdk/monitor/azure-monitor-query/README.md

```diff
@@ -405,6 +405,9 @@ raw JSON response. For example:
 }
 ```
 
+#### Overcoming Log Analytics query size limitations
+If your query exceeds the service limits, you can refer to the large log query documentation to learn how to overcome them.
```

Comment:
@srnagar I can't link to the documentation I added until the PR gets merged because it'll fail the ci. I'm going to have a follow-up linking to the documentation that is in the samples README file. I'll update this text to reflect that.